common/tsconfig.json

```diff
@@ -40,7 +40,7 @@
         // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
         // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
         // "typeRoots": [],                       /* List of folders to include type definitions from. */
-        "typeRoots": ["../verification/curator-service/ui/src/redux/@types"],
```

Comment:
I don't know enough about the TS stack to understand why it's recently changed how it resolves types, but I've seen that we had problems with jest types in the tests too. Was there a new version in the typescript compiler, or something else?